src/components/PersonCards.js

```diff
@@ -5,14 +5,15 @@ import Masonry from 'react-masonry-component'
 import { humanDate } from '../util/text'
 import { isEmpty } from 'lodash'
 
-const PersonCards = ({ people, menus, slug, onSkillClick }) => {
+const PersonCards = ({ people, menus, slug, onSkillClick, removeMember }) => {
   return <div className='person-cards masonry-container'>
     <Masonry options={{transitionDuration: 0}}>
       {people.map(person => <div key={person.id} className='masonry-item-wrapper'>
         <PersonCard person={person}
           menu={menus && menus[person.id]}
           slug={slug}
           onSkillClick={onSkillClick}
+          removeMember={removeMember}
```

Comment:
I didn't use the `menus` prop, but it doesn't make sense to have them both there. I can rewrite to use the menus prop, or just remove it. What do you think?